spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -1009,26 +1009,36 @@ example, to indicate that an LRA should automatically cancel after 100 milliseco
       @HeaderParam(LRA.LRA_HTTP_CONTEXT_HEADER) URI lraId) {...}
 ----
 
-Furthermore, the ability to compensate or complete may be transient capabilities of a
-service so participants can also be timed out. When the time limit is reached
-the LRA is cancelled and participants will be notified via their compensation
-(i.e. the method annotated with `@Compensate`). To set such a time limit add the
-`timeLimit` and `timeUnit` element to the `@Compensate` and `@Complete`
-annotations. If different time limits are set on these two methods then the
-earliest of the two will take effect. An example of how a participant could
-indicate that its' ability to compensate is limited to 100 milliseconds could be:
+When the timeout is reached, the LRA moves to the state `Cancelling` and all registered participants receive a `Compensate`
+call. Just as if the Long Running Action would be closed due to an error.
```

Comment:
I guess this is about the terminology used by the people (and thus why a terminology section in the beginning would be useful)
- Closing LRA -> Ending/Stopping the LRA (no longer active)
- Completing LRA -> successful closing LRA and @Complete will be called
- Cancelling LRA -> Since we can only cancel the LRA today by having an HTTP status 4xx or 5xx (by default, configurable) this means that an LRA is cancelled only after an 'error' occurred.

I reworded the last sentence and no longer used 'error'.